src/lib/types.ts

```diff
@@ -77,6 +77,9 @@ export interface CapabilityCfg {
 export interface CapabilityExport extends CapabilityCfg {
   bindings: Binding[];
   hasSchedule: boolean;
+  apiGroups?: string[];
```

Comment:
shouldn't this be RBAC and maybe an array of PolicyRules?

Reply:
Updated to:

```ts
export interface CapabilityExport extends CapabilityCfg {
  bindings: Binding[];
  hasSchedule: boolean;
  rbac?: PolicyRule[];
}
```